src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/mapping/CollectionMapper.kt

```diff
@@ -26,7 +26,7 @@ class CollectionMapper {
                 obj.titleKey,
                 obj.slug,
                 obj.sort,
-                metadataFk = obj.resourceContainer?.id
+                dublinCoreFk = obj.resourceContainer?.id
```

Comment:
@sarabiaj This might not have been necessary, do you agree with the change though?